content.ts

```diff
@@ -10,6 +10,9 @@ export type ContentCommandType =
   | "user-edit"
   | "revert";
 
+export type ContentStatus = "DRAFT" | "IN_REVIEW" | "APPROVED" | "PUBLISHED";
```

Comment:
Might just be me, but feels like there is something odd about forcing consumers of the SDK to specify these values in all caps, like it leaks something from our database. Do you think we could make this friendlier?

Reply:
I was thinking about exposing an enums of sorts. So you can do `ContentStatus.InReview`

Reply:
I like that!